lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -90,6 +90,7 @@ my %URL_SPECIALIZATIONS = (
     'Rockipedia'          => qr{^https?://(?:www.)?rockipedia.no/}i,
     'Rolldabeats'         => qr{^https?://(?:www.)?rolldabeats.com/}i,
     'SecondHandSongs'     => qr{^https?://(?:www.)?secondhandsongs.com/}i,
+    'SetlistFM'           => qr{^https?://(?:www.)?setlist.fm/}i,
```

Comment:
This has probably not been tested, because this module can not be found due to differences in capitalization.
Perl module names is case sensitive, thus the module name does not match the filename.
Please fix the capitalization in the name of the below added file.

Here is a test artist: https://musicbrainz.org/artist/43b58c98-3779-4b04-9a23-1c95cca3a145